controllers/search_controller.go

```diff
@@ -193,6 +202,17 @@ func (r *SearchReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *SearchReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	searchPods := predicate.Funcs{
```

Comment:
@SherinV We can remove this ? Because we reconcile for all events ?

Reply:
Done